src/hooks/useGetLogEntries.tsx

```diff
@@ -0,0 +1,32 @@
+import { ActivityLogApiGetLogEntriesRequest } from '@jellyfin/sdk/lib/generated-client';
+import type { AxiosRequestConfig } from 'axios';
+import { getActivityLogApi } from '@jellyfin/sdk/lib/utils/api/activity-log-api';
+import { useQuery } from '@tanstack/react-query';
+import { JellyfinApiContext, useApi } from './useApi';
+
+const fetchGetLogEntries = async (
+    currentApi: JellyfinApiContext,
+    requestParams: ActivityLogApiGetLogEntriesRequest,
+    options?: AxiosRequestConfig
+) => {
+    const { api } = currentApi;
+
+    if (!api) return;
+
+    const response = await getActivityLogApi(api).getLogEntries(requestParams, {
+        signal: options?.signal
+    });
+
+    return response.data;
+};
+
+export const useGetLogEntires = (
```

Comment:
Let's drop the "Get" portion of the file/function names.

```suggestion
export const useLogEntires = (
```